code/style.css

```diff
@@ -90,12 +94,12 @@ main {
 }
 
 .bubble {
-  background: #e5e9ff;
+  background: #c2334d;
   font-weight: 600;
   font-size: 16px;
   line-height: 26px;
   padding: 16px 24px;
-  color: #0026ff;
+  color: #fff;
   max-width: 40%;
 }
```

Comment:
When I put my e-mail, it overflows the bubble because it was a long one. maybe consider adding word-wrap here to prevent this unwanted behaviour. https://www.w3schools.com/cssref/css3_pr_word-wrap.asp 😺